cypress/e2e/main.cy.js

```diff
@@ -30,14 +28,70 @@ describe('Main Page', () => {
 
     cy.get('[data-cy="movie-poster"] img').last().should('have.attr', 'alt', 'Pulp Fiction poster');
     cy.get('[data-cy="vote-container"]').last().should("exist")
-
-    cy.get('[data-cy="upvote-button"]').first().click()
-    // cy.get(".voteCount").should("contain", onIncreaseVote)
   })
 
-  // it('Increment the vote count')
+  it('updates vote count with up and down buttons', () => {
```

Comment:
This stubbing of network requests was great to figure out with you, tests look great.